tensorflow_datasets/core/custom_dataset/__init__.py

```diff
@@ -0,0 +1,22 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Custom Datasets APIs"""
+
+from tensorflow_datasets.core.custom_dataset.image_folder import ImageLabelFolder
```

Comment:
I would wrap this inside `with core.registered.skip_registration()` so that `image_folder` cannot be created from `tffds.load` (has to be explicitly defined)